docs/sources/guides/whats-new-in-v7-2.md

```diff
@@ -23,6 +23,9 @@ The main highlights are:
 - [**Table column filters**]({{< relref "#table-column-filters" >}})
 - [**New and improved transforms**]({{< relref "#new-and-improved-transforms" >}})
 - [**Field override matchers**]({{< relref "#field-override-matchers" >}})
+- [**Query inspector in Explore**]({{< relref "#query-inspector-in-explore" >}})
```

Comment:
Just a reminder, every feature should have a link to updated documentation.

Reply:
Added